backend/prompt_studio/prompt_studio/helper.py

```diff
@@ -6,27 +6,61 @@
 
 
 class PromptStudioHelper:
+    @staticmethod
+    def get_prompt_model(is_sps: bool):
+        """Dynamically import and return the appropriate prompt model.
+
+        Args:
+            is_sps (bool): Flag indicating whether to use the SPSPrompt model.
+
+        Returns:
+            model (class): The appropriate prompt model class
+            (ToolStudioPrompt or SPSPrompt).
+
+        Raises:
+            ImportError: If the SPSPrompt model could not be imported.
+        """
+        if is_sps:
+            try:
+                from pluggable_apps.simple_prompt_studio.sps_prompt.models import (
+                    SPSPrompt,
+                )
+
+                return SPSPrompt
+            except ImportError as e:
+                logger.error("SPSPrompt model could not be imported.")
+                raise ImportError("SPSPrompt model could not be imported.") from e
```

Comment:
@tahierhussain we're practically catching an ImportError and re-raising the same. I don't see any benefit of doing this except for logging which will anyway be done by the middleware. Is this actually needed?

Reply:
@chandrasekharan-zipstack I've made the necessary change. FYI, I wasn't fully aware that this would be handled in the middleware. Thanks for the catch!

Reply:
@chandrasekharan-zipstack I have made modifications to this file. As per our earlier discussion, I am now passing the model class from both the main and simple prompt studio views files. This approach eliminates the need for the block of code where we were conditionally retrieving the model class.